src/coreclr/build-test.cmd

```diff
@@ -636,102 +639,35 @@ may help to copy its "DIA SDK" folder into "%VSINSTALLDIR%" manually, then try a
 exit /b 1
 
 :PrecompileFX
-set __TotalPrecompiled=0
-set __FailedToPrecompile=0
-set __FailedAssemblies=
-set __CompositeOutputDir=%CORE_ROOT%\composite.out
-set __CompositeResponseFile=%__CompositeOutputDir%\framework-r2r.dll.rsp
-
-if defined __CompositeBuildMode (
-    mkdir !__CompositeOutputDir!
-    echo --composite>>!__CompositeResponseFile!
-    echo -O>>!__CompositeResponseFile!
-    echo --out^:%__CompositeOutputDir%\framework-r2r.dll>>!__CompositeResponseFile!
-)
-
-for %%F in ("%CORE_ROOT%\System.*.dll";"%CORE_ROOT%\Microsoft.*.dll";%CORE_ROOT%\netstandard.dll;%CORE_ROOT%\mscorlib.dll) do (
-    if not "%%~nxF"=="Microsoft.CodeAnalysis.VisualBasic.dll" (
-    if not "%%~nxF"=="Microsoft.CodeAnalysis.CSharp.dll" (
-    if not "%%~nxF"=="Microsoft.CodeAnalysis.dll" (
-    if not "%%~nxF"=="System.Runtime.WindowsRuntime.dll" (
-        if defined __CompositeBuildMode (
-            echo %%F>>!__CompositeResponseFile!
-        ) else (
-            call :PrecompileAssembly "%%F" %%~nxF __TotalPrecompiled __FailedToPrecompile __FailedAssemblies
-            echo Processed: !__TotalPrecompiled!, failed !__FailedToPrecompile!
-        )
-    )))))
-)
-
-echo Composite response line^: %__CompositeResponseFile%
-type "%__CompositeResponseFile%"
+set __CompileFrameworkCmd="%CORE_ROOT%\corerun" "%CORE_ROOT%\R2RTest\R2RTest.dll"
```

Comment:
You can use `"%__RepoRootDir%\dotnet.cmd"` instead of `corerun` so we run on a release runtime.

Reply:
I'll be happy to once I manage to make the change work, thanks for pointing that out :-).